virttest/qemu_vm.py

```diff
@@ -3045,6 +3045,20 @@ def get_pid(self):
         except (TypeError, IndexError, ValueError):
             return None
 
+    def get_qemu_threads(self):
+        """
```

Comment:
you can list tasks launched by qemu by command
"ls /proc/%d/task"  % self.get_pid()